cypress/page/menu-content.page.ts

```diff
@@ -3,7 +3,7 @@ class MenuContentPage {
     private menuContentPageURL: string
 
     constructor() {
-        this.tShirtMenu = "#block_top_menu > ul > li:nth-child(3) > a";
+        this.tShirtMenu = ".sf-menu > :nth-child(3) > a";
```

Comment:
the key here, is not depend on the position of any item, you can preserve the beginning of the old one `#block_top_menu > ul > li` but then look for an attribute of the `a` tag that can be used to indicate that we are selecting the t-shirts option